cg/services/order_validation_service/rules/case_sample/utils.py

```diff
@@ -22,3 +51,252 @@ def get_invalid_panels(panels: list[str], store: Store) -> list[str]:
         panel for panel in panels if not store.does_gene_panel_exist(panel)
     ]
     return invalid_panels
+
+
+def is_volume_invalid(sample: Sample) -> bool:
+    in_container: bool = is_in_container(sample.container)
+    allowed_volume: bool = is_volume_within_allowed_interval(sample.volume)
+    return in_container and not allowed_volume
+
+
+def is_in_container(container: ContainerEnum) -> bool:
+    return container != ContainerEnum.no_container
+
+
+def is_volume_within_allowed_interval(volume: int) -> bool:
+    return MINIMUM_VOLUME <= volume <= MAXIMUM_VOLUME
```

Comment:
These exist in order_validation_service/rules/utils.py

Reply:
https://github.com/Clinical-Genomics/cg/pull/3625/commits/03d1b7263aab6bb51e2a44ac7d2743115b542ff4